### PR TITLE
Alpha 0.1.7/329 write tests for comportpy

### PR DESCRIPTION
**Pull request by Jopat2409**

### CODE REVIEW

File: radio/app/drone.py [hunk 1]
```diff
@@ -146,6 +176,13 @@ def __getNextLogFilePath(self, line: str) -> str:
     def __getCurrentDateTimeStr(self) -> str:
         return time.strftime("%Y-%m-%d_%H-%M-%S", time.localtime())
 
+    def checkBaudrateValid(self, baud: int) -> bool:
```
Comment (NexInfinite) [inline radio/app/drone.py:179]: this should be also a static method

**Review by NexInfinite — CHANGES_REQUESTED** [review]

File: radio/app/drone.py (hunk 1 above)
Comment (Jopat2409) [reply radio/app/drone.py:179]: epic

Comment (NexInfinite) [reply radio/app/drone.py:179]: ROFL!

**Review by NexInfinite — APPROVED** [review]
LGTM

**Pull request merged**